bridge/src/lib.rs

```diff
@@ -216,8 +221,27 @@ pub mod pallet {
 	/// Mark whether a deposit nonce was used. Used to mark execution status of a proposal.
 	#[pallet::storage]
 	#[pallet::getter(fn used_nonces)]
-	pub type UsedNonces<T: Config> =
-		StorageMap<_, Twox64Concat, DepositNonce, DepositNonce, ValueQuery>;
+	pub type UsedNonces<T: Config> = StorageDoubleMap<
+		_,
+		Twox64Concat,
+		DomainID,
+		Twox64Concat,
+		DepositNonce,
+		DepositNonce,
+		ValueQuery,
+	>;
+
+	/// Mark supported dest domainID
+	#[pallet::storage]
+	#[pallet::getter(fn dest_domain_ids)]
+	pub type DestDomainIds<T: Config> = StorageMap<_, Twox64Concat, DomainID, bool, ValueQuery>;
+
+	/// Mark the pairs for supported dest domainID with its corresponding chainID
+	/// The chainID is not directly used in pallet, this map is designed more about rechecking the
+	/// domainID
+	#[pallet::storage]
+	#[pallet::getter(fn dest_chain_ids)]
+	pub type DestChainIds<T: Config> = StorageMap<_, Twox64Concat, DomainID, ChainID, ValueQuery>;
```

Comment:
```suggestion
	pub type DestChainIds<T: Config> = StorageMap<_, Twox64Concat, DomainID, ChainID>;
```

If we don't set `ValueQuery` here, there will be no default value `0` returned if no dest chainid is bound to the dest domainid. I think it's more reasonable to return `None` rather than `0` if we haven't set the value.

Reply:
yes of course it's better, I was thinking that since there is no evm chain has 0 as ChainID, but set the default value to `None` make more sense